### PR TITLE
Logging configuration fix

### PR DESCRIPTION
**Pull request by azax42**
Hello, my company has adopted your package for our internal use and I have made some enhancements that I think might be of use outside of our company.

This PR allows adjusting the log level via a cli switch or environment variable, cleans up the log message format and fixes an ambiguous log message 

If you are willing to accept this PR (or PRs in general) I plan to submit a few more PRs of roughly similar size and scope

### CODE REVIEW

File: src/clickhouse_migrations/cmd.py [hunk 1]
```diff
@@ -13,6 +14,11 @@
     MIGRATIONS_DIR,
 )
 
+def log_level(value: str) -> str:
+    if value.upper() in ("DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"):
```
Comment (zifter) [inline src/clickhouse_migrations/cmd.py]: maybe it's better to use logging._nameToLevel.keys()?
yes, it's private variable, but it contains all levels and aliases.

**Review by zifter — CHANGES_REQUESTED** [review]

**Comment by zifter** [comment]
You need to add small unit test, which uses log_level :)
I'm trying to keep 100% coverage.

**Comment by zifter** [comment]
@azax42 Thank you very much for your PR!
I'm ready to accept, review and discuss any changes to this repo, so feel free to make more PR! :)

**Review by zifter — APPROVED** [review]

File: src/tests/test_log_level.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        level_char_list = []
+        for i, ch in enumerate(level):
+            ch = chr(ord(ch) | 0b0010_0000) if (i % 2) != 0 else ch
+            level_char_list.append(ch)
+        mixed_case_level = "".join(level_char_list)
+        assert log_level(mixed_case_level) == level
+
+
+def test_invalid_log_levels():
+    bad_input = (
```
Comment (zifter) [inline src/tests/test_log_level.py:34]: it's good to use parameterized for table tests :)

Comment (azax42) [reply src/tests/test_log_level.py:34]: Not sure I understand?

**Pull request merged**